build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
 //
 // It must be false for fat aar generation of MuxExoPlayer to work
 // This is a bug/problem with the fat aar mechanism
-def inLocalDevModeFlag = false
```

Comment:
For the saucelabs testing etc. to work this needs to be false. (This is because the testing phase depends on the fat aar pushed to the dev maven repo in the first phase).

(Meta comment) This flag is a real pain, and I think we should actively try to think of better ways around it - ideally by fixing the fat aar lib so that embed behaves as expected across modules.

Reply:
(Meta comment) Interesting... I'll think about alternatives.